modules/transport.js

```diff
@@ -518,7 +523,31 @@ Transport.prototype.onNewBlock = function (block, broadcast) {
  */
 Transport.prototype.onMessage = function (msg, broadcast) {
 	if (broadcast && !__private.broadcaster.maxRelays(msg)) {
-		__private.broadcaster.broadcast({limit: constants.maxPeers, dappid: msg.dappid}, {api: '/dapp/message', data: msg, method: 'POST', immediate: true});
+		__private.broadcaster.broadcast({limit: constants.maxPeers, dappid: msg.dappid}, {
+			api: '/dapp/message',
+			data: msg,
+			method: 'POST',
+			immediate: true
+		});
+	}
+};
+
+/**
+ * Broadcasts the passed block using __private.broadcaster.broadcast
+ * @implements {Broadcaster.broadcast}
+ * @param {Object} params
+ * @param {Object} block
+ */
+Transport.prototype.broadcastBlock = function (params, block) {
+	if (!__private.broadcaster.maxRelays(block)) {
+		library.logger.info('Broadcasting the block of id: ' + block.id);
+		library.logger.trace('Broadcasting the block parameters', + params);
```

Comment:
It's `trace` level - maybe log both parameters and block object?
`library.logger.trace('Transport->broadcastBlock', {block: block, params: params});`